Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -120,11 +120,15 @@
 )
 globalValidationTrackingOnly = cms.Sequence()
 
-def _modifyGlobalValidationForPhase2( theProcess ):
-    theProcess.load('Validation.Configuration.gemSimValid_cff')
-    theProcess.load('Validation.Configuration.me0SimValid_cff')
-    theProcess.globalValidation += theProcess.gemSimValid
-    theProcess.globalValidation += theProcess.me0SimValid
+from Validation.Configuration.gemSimValid_cff import *
+from Validation.Configuration.me0SimValid_cff import *
+
+_run3_globalValidation = globalValidation.copy()
+_run3_globalValidation += gemSimValid
+
+_phase2_globalValidation = _run3_globalValidation.copy()
+_phase2_globalValidation += me0SimValid
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesGlobalValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyGlobalValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( globalValidation, _run3_globalValidation )
+eras.phase2_muon.toReplaceWith( globalValidation, _phase2_globalValidation )
```

Comment:
@makortel The order is the other way around. @slava77 had `Phase2 = (phase2_muon, run3_GEM)` so `phase2_muon` gets run first then `run3_GEM` would be run. However that would do the opposite of what @slava77 said. Instead the changes done by `phase2_muon` would be lost so `globalValidation` would only contain the changed done by `run3_GEM` and it would look like `phase2_muon` never did anything to `globalValidation`.

This also means that if one ran `phase2_muon` without also running `run3_GEM` then `globalValidation` would still contain `gemSimValid`!